README.md

```diff
@@ -1,2 +1,4 @@
 # pr_observation_repo
 just a trial.
+
```

Comment:
unnecessary new line. Can you remove it?

Reply:
It is necessary for bla bla.